_bibliography/pint.bib

```diff
@@ -678,12 +742,26 @@ @article{MadayTurinici2003
 	year = {2003}
 }
 
+@article {SheenEtAl2003,
+	author = {Sheen, Dongwoo and Sloan, Ian H. and Thom\'{e}e, Vidar},
+	title = {{A parallel method for time discretization of parabolic equations based on {L}aplace transformation and quadrature}},
+	journal = {IMA J. Numer. Anal.},
+	fjournal = {IMA Journal of Numerical Analysis},
```

Comment:
Could you please remove fjournal and use the full name in journal? Just to be consistent with the other entries.